torch_np/tests/test_ufuncs_basic.py

```diff
@@ -105,12 +105,6 @@ def test_x_and_out_broadcast(self, ufunc):
     (np.add, operator.__add__, operator.__iadd__),
     (np.subtract, operator.__sub__, operator.__isub__),
     (np.multiply, operator.__mul__, operator.__imul__),
-    (np.divide, operator.__truediv__, operator.__itruediv__),
-    (np.floor_divide, operator.__floordiv__, operator.__ifloordiv__),
-    (np.float_power, operator.__pow__, operator.__ipow__),
-    ##   (np.remainder, operator.__mod__, operator.__imod__),   # does not handle complex
-    # remainder vs fmod?
-    # pow vs power vs float_power
```

Comment:
What happened to these?

Reply:
It's just that the smoke test parametrized on these is too simplistic to catch that `integer / integer -> float` (algebraic fields vs rings?). More comprehensive testing of these things is in `test_ufuncs` in numpy, and porting it is on a TODO list.

Since these all are now generated by the same loop, the parametrization is really not essential I think.

Reply:
fair enough